komga/src/main/kotlin/org/gotson/komga/interfaces/api/rest/BookController.kt

```diff
@@ -543,6 +543,19 @@ class BookController(
     }
   }
 
+  @DeleteMapping("api/v1/books/{bookId}")
```

Comment:
I am conflicted about that one. We are not deleting the Book entity, but the underlying file.

I think it would make more sense to use `api/v1/books/{bookId}/file` as the path instead.